pkg/cmd/stack/deploy.go

```diff
@@ -165,7 +165,7 @@ func deploy(ctx context.Context, s *model.Stack, c *kubernetes.Clientset, config
 	case <-stop:
 		log.Infof("CTRL+C received, starting shutdown sequence")
 		spinner.Stop()
-		os.Exit(130)
+		return fmt.Errorf("interrupt signal received")
```

Comment:
I think we do `os.Exit(130)` in many other places to handle "CTRL+C". Should we update it everywhere?

Reply:
I thought is was better to use `os.Exit(130)` to follow terminal rules but returning an error is better for debugging on remotes